spec/spec_helper.rb

```diff
@@ -24,6 +24,7 @@
   config.include ApplicationHelper
   config.include LoginHelpers
   config.include JobrefHelpers
+  config.include ActiveSupport::Testing::TimeHelpers
```

Comment:
happy to add this, however just wondering, what value does it add to this PR or the existing tests?

Reply:
Allows you to use [TimeHelpers](https://api.rubyonrails.org/v5.2.4.2/classes/ActiveSupport/Testing/TimeHelpers.html#method-i-travel) which are useful when testing TimeZone aware code. [I used `travel` in the code here - I like the block form which only changes the time within the block](https://github.com/codebar/planner/pull/1311/commits/46b23fc2be3f7b67fcca60d19dc35eab43da5ae7#diff-0852c341dc1026ff5fd36822b4b82433R33)

[My idea would be to remove any reference to TimeCop](https://andycroll.com/ruby/replace-timecop-with-rails-time-helpers-in-rspec/)

Reply:
Nice, I didn't notice the reference to `travel`. Removing TimeCop if there is an ActiveSupport alternative sounds great. I couldn't even remember that we were using it :)